portfolio/src/main/webapp/style.css

```diff
@@ -1,43 +1,138 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  overflow: hidden;
+  margin-top: 5%;
+  width: 600px;
+  height: 500px;
 }
 
 #image-container {
-  margin-top: 20px;
+  object-fit: cover;
+  border: 2px black;
+  width: 70%;
+  height: 70%;
 }
 
 body,html {
     background-image: url("images/ghibli.jpg");
-    background-repeat: repeat-y;
+    background-repeat: no-repeat;
     background-attachment: fixed;
     height:100%;
+    margin: 0;
+    font-family: Georgia, Helvetica, sans-serif;
+    color: white;
+    letter-spacing: 0.1em;
+    scroll-behavior: smooth;
+}
+
+.sidenav {
+  width: 150px;
+  position: fixed;
+  z-index: 1;
+  top: 28.8%;
+  left: 10px;
+  background: rgb(0,0,0,0);
+  overflow-x: visible;
+  padding: 8px 0;
+}
+
+.sidenav a {
+  padding: 6px 0px 50px 0px;
+  text-decoration: none;
+  font-size: 14pt;
+  font-weight: bold;
+  color: white;
+  display: block;
+}
+
+.sidenav a:hover {
+  color: rgb(211,211,211);
+}
+
+@media screen and (max-height: 450px) {
+  .sidenav {padding-top: 50px;}
+  .sidenav a {font-size: 28px;}
+}
+
+.button {
+  background-color: rgba(255, 255, 255, 0.384);
+  border: 2px black;
+  border-radius: 8px;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  font-family: Georgia;
+  letter-spacing: 0.1em;
+  margin: 4px 2px;
+  cursor: pointer;
+}
+
+.button: hover{
+    color: rgb(211,211,211);
+}
+
+.arrow {
+  border: solid white;
+  border-width: 0 2px 2px 0;
+  border-radius: 2px;
+  display: inline-block;
+  padding: 4px;
+}
+
+.up {
+  transform: rotate(-135deg);
+  -webkit-transform: rotate(-135deg);
+}
+
+#top-button {
+  display: none;
+  position: fixed;
+  bottom: 20px;
+  right: 2%;
+  z-index: 99;
+  font-size: 13pt;
+  font-family:Georgia;
+  font-weight: bold;
+  border: none;
+  outline: none;
+  background-color: rgba(0,0,0,0);
+  color: white;
+  cursor: pointer;
+  padding: 15px;
+  border-radius: 8px;
+}
+
+#top-button:hover {
+  color: rgb(211,211,211);
 }
 
 .bg-text {
-  background: rgba(0,0,0,0.3);
+  background: rgba(0,0,0,0.35);
   color: white;
   font-weight: bold;
   line-height:1.6;
   border: 3px solid #f1f1f1;
   position: absolute;
-  top: 150%;
+  top: 230%;
   left: 50%;
   transform: translate(-50%, -50%);
```

Comment:
What's the reason to use transform? We can use margin-top/bottom/right/left to settle the element position as well.